projects/driver_controls/src/ads1015.c

```diff
@@ -0,0 +1,157 @@
+#include "ads1015.h"
+
+#include <stdbool.h>
+#include <stdio.h>
+
+#include "i2c.h"
+#include "delay.h"
+#include "gpio_it.h"
+#include "ads1015_bitmasks.h"
+
+// Union definition to help process registers data
```

Comment:
Is this comment about the `union` still valid? You no longer have a `union`.

Reply:
It isn't, I'll get that changed